bindsnet/datasets/torchvision_wrapper.py

```diff
@@ -31,8 +31,8 @@ class torchvision_dataset_wrapper(ds_type):
 
         def __init__(
             self,
-            image_encoder: Optional[Encoder],
-            label_encoder: Optional[Encoder],
+            image_encoder: Optional[Encoder] = None,
+            label_encoder: Optional[Encoder] = None,
```

Comment:
I don't think this can be `None` as the order of the arguments matter in this case. The point is to split the requirements. First we take care of Bindsnet requirements `image_encoder` and `label_encoder` then we take care of specific dataset requirements `*args` and `**kwargs`. I think forcing these (even if somebody puts `None`) keeps the consistency. @djsaunde thoughts?